core/src/main/scala/akka/persistence/r2dbc/internal/BySliceQuery.scala

```diff
@@ -331,7 +333,8 @@ import org.slf4j.Logger
     }
 
     def switchFromBacktracking(state: QueryState): Boolean = {
```

Comment:
Interesting, is this backtracking somewhere documented? Or only in the code?

Reply:
Some is explained in https://github.com/akka/akka-persistence-r2dbc/blob/main/design-notes.md